src/android/CordovaCall.java

```diff
@@ -197,6 +199,7 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
                 pendingAction = "receiveCall";
                 this.checkCallPermission();
             }
+            pendingAction = null;
```

Comment:
just a question about logic about this unconditional reset.

`checkCallPermission` could return early if `READ_PHONE_NUMBERS` is not granted. can we assume that when the `READ_PHONE_NUMBERS` is granted, the app would be restarted by the os?

Reply:
oh that's a good point, because If its not restarted, the action would still be pending

checking...

Reply:
Yes it seems that if the "phone" permission is changed the app gets restarted

Reply:
am going to remove this logic clearing pendingAction